README.md

```diff
@@ -40,9 +40,24 @@ $ ./startup.sh
 ```
 If you are using the GDS development virtual machine then the application will be available on the host at https://specialist-publisher-rebuild.dev.gov.uk/
 
+## Granting Permissions
+
+You may find that you can't see any documents after replicating data. To fix
+this, you need to grant user permissions in this application:
+
+```bash
+bundle exec rake permissions:grant['Chris Patuzzo'] # Or whichever user you're logged in as.
+```
+
+You also need to set the `app_name` for the Dummy API User in Publishing API:
+
+```ruby
+User.find_by(email: "dummyapiuser@domain.com").update!(app_name: "specialist-publisher")
```

Comment:
Can we consolidate this blocks of text with line 65 - 76? https://github.com/alphagov/specialist-publisher-rebuild/pull/802/files#diff-04c6e90faac2675aa89e2176d2eec7d8R74